### PR TITLE
fix(launch): add load option to docker buildx

### PR DESCRIPTION
**Pull request by KyleGoyette**
Description
-----------

- [Fixes WB-16152](https://wandb.atlassian.net/browse/WB-16152)
- Fixes #NNNN

What does the PR do?

### <samp>🤖 Generated by Copilot at f012236</samp>

Fixed a bug in `wandb/docker/__init__.py` that prevented the wandb docker image from being pushed to the registry. Added the `--load` flag to the `buildx` command to load the image into the local docker daemon.

Testing
-------
How was this PR tested?

### <samp>🤖 Generated by Copilot at f012236</samp>

> _`buildx` needs `--load`_
> _to push wandb docker image_
> _a winter bug fix_

### CODE REVIEW

File: tests/pytest_tests/unit_tests/test_docker.py [hunk 1]
```diff
@@ -20,3 +20,12 @@ def mock_shell():
 def test_buildx_not_installed(runner):
     with runner.isolated_filesystem():
         assert is_buildx_installed() is False
+
+
+@pytest.mark.parametrize(
+    "platform,adds_load_arg",
+    [(None, True), ("linux/amd64", True), ("linux/amd64,linux/arm664", False)],
```
Comment (nickpenaranda) [inline tests/pytest_tests/unit_tests/test_docker.py:27]: typo: `arm664`

File: wandb/docker/__init__.py [hunk 2]
```diff
@@ -114,6 +116,12 @@ def build(
     return stdout
 
 
+def should_add_load_argument(platform: Optional[str]) -> bool:
```
Comment (nickpenaranda) [inline wandb/docker/__init__.py:119]: Can you leave a comment here?  Failing to understand why the platform parameter determines the need for this

**Review by nickpenaranda — APPROVED** [review]

File: tests/pytest_tests/unit_tests/test_docker.py (hunk 1 above)
Comment (KyleGoyette) [reply tests/pytest_tests/unit_tests/test_docker.py]: fixed

File: wandb/docker/__init__.py (hunk 2 above)
Comment (KyleGoyette) [reply wandb/docker/__init__.py:119]: Added

**Pull request merged**